packages/cli/commands/init.js

```diff
@@ -101,15 +103,18 @@ exports.handler = async options => {
 
   trackAuthAction('init', authType, TRACKING_STATUS.STARTED);
 
+  if (c) {
+    setConfigPath(path.join(process.cwd(), c));
```

Comment:
Instead of calling `setConfigPath()` here, what if `createEmptyConfigFile` accepted a path? I think we also should use the `getCwd()` helper instead of `process.cwd()` so that we're consistent when it comes to how we're resolving CWD.

Reply:
I removed this code.